core/domain/user_services.py

```diff
@@ -669,7 +787,7 @@ def _get_user_settings_from_model(user_settings_model):
     )
 
 
-def is_user_registered(user_id):
+def is_user_registered(user_id: Optional[str]) -> bool:
```

Comment:
Should we update the docstring to cover the None case?

(Or alternatively fix the callsites to always pass in a non-None value?)

Reply:
For this case call sites are fixed!